src/meta/permissions.rs

```diff
@@ -127,6 +127,7 @@ impl Permissions {
         ColoredString::new(Colors::default_style(), res)
     }
 
+    #[cfg_attr(windows, allow(dead_code))]
```

Comment:
Nit, we can change this to `#[cfg(not(windows))]`.

Reply:
done :innocent: